src/components/previewCard/imageHelper.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from "react"
+
+import styles from "./index.module.css"
+
+interface Props {
+  imageSrc: string
+  pictureAlt: string
+  isLogo: boolean
+  handleOnErrorImage?: (e) => void
+}
+
+const Image: FC<Props> = ({
```

Comment:
please rename the file to `image.tsx`, so it's in line with the naming of the component